handlers.go

```diff
@@ -23,8 +23,9 @@ import (
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
 
-	"github.com/fullstorydev/grpcui/internal"
 	"github.com/fullstorydev/grpcurl"
+
+	"github.com/fullstorydev/grpcui/internal"
```

Comment:
Is this separation because grpcui is this module wherease grpcurl is a remote module?